src/main/resources/style.css

```diff
@@ -0,0 +1,18 @@
+.title-label{
```

Comment:
Make the style name more specific since this file is under the resources folder, which organizes the whole fxml files